examples/LoggingExtensions/Program.cs

```diff
@@ -14,14 +14,13 @@
 // limitations under the License.
 // </copyright>
 
+using System.Diagnostics.Tracing;
 using OpenTelemetry.Logs;
 using OpenTelemetry.Resources;
 using Serilog;
 
-var resourceBuilder = ResourceBuilder.CreateDefault().AddService("Examples.LogEmitter");
+var resourceBuilder = ResourceBuilder.CreateDefault().AddService("Examples.LoggingExtensions");
 
-// Note: It is important that OpenTelemetryLoggerProvider is disposed when the
-// app is shutdown. In this example we allow Serilog to do that by calling CloseAndFlush.
 var openTelemetryLoggerProvider = new OpenTelemetryLoggerProvider(options =>
```

Comment:
@alanwest @cijothomas @utpilla One thing I realized working on tests. The Serilog + EventSource APIs ask you to pass in the `OpenTelemetryLoggerProvider`. If you are creating it manually, that is super easy to do. However if you are using the `ILoggingBuilder` extension it is tricky/non-obvious how you get the actual `OpenTelemetryLoggerProvider`. Maybe as a follow-up I'll see if we can smooth that out somehow?